packages/tabs/src/Tabs.tsx

```diff
@@ -132,36 +133,43 @@ function Tabs({
     setSelected(index);
   }
 
-  function handleKeyDown(e: React.KeyboardEvent) {
-    e.stopPropagation();
-
-    const enabledIndexes = childrenArray.reduce((acc, child, index) => {
-      if (child.props.disabled) {
-        return acc;
-      }
-
-      return [...acc, index];
-    }, [] as Array<number>);
-
-    const enabledCurrentIndex = enabledIndexes.indexOf(selected!);
-
-    switch (e.keyCode) {
-      case keyMap.ArrowRight:
-        setSelected(
-          enabledIndexes[(enabledCurrentIndex + 1) % enabledIndexes.length],
-        );
-        break;
-
-      case keyMap.ArrowLeft:
-        setSelected(
-          enabledIndexes[
-            (enabledCurrentIndex - 1 + enabledIndexes.length) %
-              enabledIndexes.length
-          ],
-        );
-        break;
+  const enabledIndexLogic: () => [Array<number>, number] = () => {
+    const enabledIndexes: Array<number> = childrenArray.reduce(
+      (acc, child, index) => {
+        if (child.props.disabled) {
+          return acc;
+        }
+
+        return [...acc, index];
+      },
+      [] as Array<number>,
+    );
+
+    return [enabledIndexes, enabledIndexes.indexOf(selected!)];
+  };
+
+  const handleArrowKeyPress = e => {
+    if (e.keyCode === keyMap.ArrowRight) {
+      const [enabledIndexes, current] = enabledIndexLogic();
+      setSelected(enabledIndexes[(current + 1) % enabledIndexes.length]);
+    } else if (e.keyCode === keyMap.ArrowLeft) {
+      const [enabledIndexes, current] = enabledIndexLogic();
+      setSelected(
+        enabledIndexes[
+          (current - 1 + enabledIndexes.length) % enabledIndexes.length
+        ],
+      );
     }
-  }
+  };
+
+  const enabled =
+    navigator.platform.indexOf('Mac') > -1
+      ? !useKeyPress(keyMap.Command)
+      : !useKeyPress(keyMap.Control);
+
+  useEventListener('keydown', handleArrowKeyPress, {
```

Comment:
Feels weird to have this additional hook being used. Why not just add a handler to `useKeyPress`?

Also this feels borderline like `useKeyPress` is only ever going to be needed by Tabs, I'm not sure I see the reusability of it yet.

May make more sense to define the hook in the Tabs file and we can see if we see ourselves needing it again in the future before we export it and make it a fully supported hook.

Reply:
Not sure I understand what you mean here. (As an aside: I moved useKeyPress to tabs so its local and not external.) Happy to make the hook more robust if that's what you're getting at? But thought it made sense to reuse logic we already had.

Reply:
Nvm. Removed the hook entirely!